working/1426-extension-types/feature-specification-views.md

```diff
@@ -641,17 +647,30 @@ Let `V` be a view type of the form
 <code>View&lt;T<sub>1</sub>, .. T<sub>s</sub>&gt;</code>,
 and let `R` be the corresponding instantiated representation type.
 `V` is a proper subtype of `Object?`. If `R` is non-nullable then `V`
-is a proper subtype of `Object` as well.
+is a proper subtype of `Object` as well. 
 
 *That is, an expression of a view type can be assigned to a top type
 (like all other expressions), and if the representation type is
 non-nullable then it can also be assigned to `Object`. Non-view types
-(except bottom types) cannot be assigned to view types without a cast.*
-
-In the body of a member of a view declaration _DV_ named `View`
-and declaring the type parameters
-<code>X<sub>1</sub>, .. X<sub>s</sub></code>,
-the static type of `this` is
+(except bottom types) cannot be assigned to view types without a cast,
+except in the case mentioned below.*
+
+Let _DV_ be a view class declaration named `View` with type parameters
+<code>X<sub>1</sub>, .. X<sub>s</sub></code>.
+In the body of _DV_, the representation type is a subtype of the view type
```

Comment:
What does "body" mean? Does it affect type parameters on the view itself, or type arguments in the `extends`/`implements` clauses? Or is it only the member declarations? 
For the member declarations, does it affect their public API, or is it just inside *their* bodies?

Reply:
This subtype relation should hold for the bodies of member declarations. Clarified that.